# Diff: https://project-skyscraper.com/news-sitemap.xml
# File: discovery/news-sitemap.xml
# Timestamp: 2026-05-28 14:15:39 UTC
# Lines: 58 old -> 45 new
# Beautified: yes
--- old/discovery/news-sitemap.xml
+++ new/discovery/news-sitemap.xml
@@ -42,17 +42,4 @@
    <news:genres>Blog</news:genres>
   </news:news>
  </url>
- <url>
-  <loc>https://project-skyscraper.com/2026/05/25/sec-log-113610/</loc>
-  <lastmod>2026-05-26T21:56:59Z</lastmod>
-  <news:news>
-   <news:publication>
-    <news:name>project-skyscraper</news:name>
-    <news:language>en</news:language>
-   </news:publication>
-   <news:title>SECURITY LOG</news:title>
-   <news:publication_date>2026-05-25T10:36:00Z</news:publication_date>
-   <news:genres>Blog</news:genres>
-  </news:news>
- </url>
-</urlset>
+</urlset>